src/app/actions/admin/products.tsx

```diff
@@ -313,26 +282,39 @@ export async function editProduct(
     ) as AvailabilityStatus,
     minimumOrderQuantity: Number(formData.get("minimumOrderQuantity")),
     returnPolicy: formData.get("returnPolicy") as ReturnPolicy,
-    images: imageUrl,
-    thumbnail: imageUrl,
   };
 
   const result = productSchema.safeParse(rawData);
-
   if (!result.success) {
-    console.log(result);
     return {
       success: false,
-      message: "Error updating a new product to Firebase",
+      message: "Error updating a product to Firebase",
       inputs: { ...rawData },
       errors: result.error.flatten().fieldErrors,
     };
   }
+  const id = Date.now().toString();
+  const blobResult = await vercelBlobPutAction({
```

Comment:
Before uploading the image to blob, check if the existing image has changed. If the image wasn't changed, no need to update it in the blob.

If the image was changed, you'd need to delete the old image from the blob and upload the new one.